PasswordVerifier.cs

```diff
@@ -0,0 +1,56 @@
+﻿namespace PasswordVerifier
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {
+            Console.WriteLine("Enter your password: ");
+            string password;
+            password = Console.ReadLine();
+            bool isTrue = true;
+            try
+            {
+                if (string.IsNullOrWhiteSpace(password))
+                {
+                    isTrue = false;
+                    throw new Exception("Password should not be empty");
+
+                }
+                if (password.Length < 8)
```

Comment:
use constant for '8', like
MIN_PASS_LENGTH = 8
and the condition will look like,
`if(password.Length < MIN_PASS_LENGTH)`

Reply:
Done! Resolved on the new commit. Thank you!